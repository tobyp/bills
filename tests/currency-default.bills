#!./bills

%set XTS 2.0  # XTS is the test currency code

A	B.	4.00

A	B.	2.00*XTS

A	B.	4.00

A	B.	2.00*XTS
