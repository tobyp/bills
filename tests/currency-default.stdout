todo: B pays 16.0 to A
